src/docs/user/references/Notification-service.adoc

```diff
@@ -0,0 +1,237 @@
+=== Overview
+
+The <<_glossary_notification_portal,*Notification Portal*>> allows users to subscribe, be notified and react when different
+types of events occur during <<_glossary_job,*Job*>> execution, on the state of the <<_glossary_proactive_scheduler,*ProActive Scheduler*>> or when
+automating and scheduling jobs with the <<_glossary_job_planner,*Job Planner*>>.
+
+image::notifications-view.png[align="center"]
+
+=== Subscriptions
+
+==== Principle
+
+In order to be notified of events a user must have an active <<_glossary_notification_subscription,*Subscription*>> with an associated active notification method.
+It is possible to be notified via the <<_glossary_notification_portal,*Notification Portal*>>, email
+or <<_glossary_notification_third_party,*Third party*>> which is the execution of a script provided by the user.
+
+All these notification methods can be used simultaneously in the same <<_glossary_notification_subscription,*Subscription*>>.
+
+==== Subscription types
+
+There are three types of <<_glossary_notification_subscription,*Subscriptions*>>:
+
+- Scheduler subscription: To be notified when the <<_glossary_proactive_scheduler ,*Scheduler*>> changes its state
+- Job planner subscription: Currently this subscription can be activated with only one event type "All events"
+that corresponds to different issues the <<_glossary_job_planner,*Job Planner*>> may encounter.
+Check out the <<_job_planner_notifications,Job Planner Notifications>> to look at the full list of events.
+- Job subscriptions: To be notified of events that may occur during <<_glossary_workflow,*Workflow*>> executions.
+
+By default each user has a Scheduler and a Job Planner subscription. To start receiving notifications it is ony required
+to choose notification methods, event types and activate the <<_glossary_notification_subscription,*Subscription*>>.
+
+On the other hand Job subscriptions have to be created in order to receive notifications.
+To create a job subscription, go to the <<_glossary_notification_portal,*Notification Portal*>>,
+job subscriptions tab and click on the _+_ button
+
+image::job-subscription-creation-button.png[align="center"]
+
+Then fill and select the mandatory fields and click on _Save_
+
+image::job-subscription-creation.png[align="center"]
+
+At least one of the fields of <<_glossary_workflow,*Workflow*>> data must be provided to create the subscription, that means it
```

Comment:
At least ...., that means ....      I do not understand the relationship..

Reply:
By saying that it is mandatory to provide at least one filter in the subscription I wanted to specify that it is not possible for a user to create a "global" job subscription that will listen to every job submitted to the Scheduler